src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -105,49 +115,85 @@ const DateTimePicker = (props: Props) => {
-      </InputGroup.Prepend>
-      <DatePicker
-        className="form-control"
-        dateFormat={dateFormat}
-        dateFormatCalendar={dateFormatCalendar}
-        disabled={disabled}
-        dropdownMode={dropdownMode}
-        endDate={endDate}
-        excludeDates={excludeDates}
-        includeDates={includeDates}
-        inline={inline}
-        locale={locale}
-        maxDate={maxDate}
-        maxTime={maxTime}
-        minDate={minDate}
-        minTime={minTime}
-        monthsShown={monthsShown}
-        onChange={onChange}
-        selected={selected}
-        selectsEnd={selectsEnd}
-        selectsStart={selectsStart}
-        showMonthDropdown={showMonthDropdown}
-        showTimeSelect={showTimeSelect}
-        showYearDropdown={showYearDropdown}
-        showTimeSelectOnly={showTimeSelectOnly}
-        timeFormat={timeFormat}
-        timeIntervals={timeIntervals}
-        timeCaption={timeCaption}
-        todayButton={todayButton}
-        startDate={startDate}
-        withPortal={withPortal}
-      >
-        {children}
-      </DatePicker>
-    </InputGroup>
+    <div>
+      <InputGroup className={className}>
+        <InputGroup.Prepend>
+          <InputGroup.Text>
+            <Icon icon="calendar" />
+          </InputGroup.Text>
+        </InputGroup.Prepend>
+        <DatePicker
+          className={`form-control ${
+            isInvalid ? 'invalid-date-picker' : isValid ? 'valid-date-picker' : ''
+          }`}
+          dateFormat={dateFormat}
+          dateFormatCalendar={dateFormatCalendar}
+          disabled={disabled}
+          dropdownMode={dropdownMode}
+          endDate={endDate}
+          excludeDates={excludeDates}
+          includeDates={includeDates}
+          inline={inline}
+          locale={locale}
+          maxDate={maxDate}
+          maxTime={maxTime}
+          minDate={minDate}
+          minTime={minTime}
+          monthsShown={monthsShown}
+          onChange={onChange}
+          selected={selected}
+          selectsEnd={selectsEnd}
+          selectsStart={selectsStart}
+          showMonthDropdown={showMonthDropdown}
+          showTimeSelect={showTimeSelect}
+          showYearDropdown={showYearDropdown}
+          showTimeSelectOnly={showTimeSelectOnly}
+          timeFormat={timeFormat}
+          timeIntervals={timeIntervals}
+          timeCaption={timeCaption}
+          todayButton={todayButton}
+          startDate={startDate}
+          withPortal={withPortal}
+        >
+          {children}
+        </DatePicker>
+      </InputGroup>
+      {isValid && validFeedback && (
+        <div className="text-left small ml-5 mt-1 text-success">
+          <svg
+            className="success-filter"
+            xmlns="http://www.w3.org/2000/svg"
+            width="14"
+            height="14"
+            viewBox="0 0 24 24"
+          >
+            <path d="M20.285 2l-11.285 11.567-5.286-5.011-3.714 3.716 9 8.728 15-15.285z" />
+          </svg>
```

Comment:
We may want to abstract this to it's own component.

Reply:
Not too sure how we would approach this and considering it has design decisions I left the core validation styling and will leave this additional feature for another issue